Tests/Library/Client.tests.ts

```diff
@@ -484,9 +483,9 @@ describe("Library/Client", () => {
 
                 assert.equal(obj0.baseType, "Microsoft.ApplicationInsights.RemoteDependencyData");
                 assert.equal(obj0.baseData.success, false);
-                assert.equal(obj0.baseData.value, 10);
-                assert.equal(obj0.baseData.name, "GET http://bing.com/search");
-                assert.equal(obj0.baseData.commandName, "http://bing.com/search?q=test");
+                assert.equal(obj0.baseData.duration, "00:00:00.010");
```

Comment:
Why is duration a string instead of an int?
I can see formatting like this for display purposes but I don't think storing the data like this is a good idea.  Any calculations now have to parse the string into an int first which will be horribly inefficient in Kusto or elsewhere.